README.adoc

```diff
@@ -13,6 +13,22 @@ If you are not on macOS or you don't have Homebrew, you need to install:
 - `protoc-gen-gogoslick` toolchain
 - `jq`
 
+Regardless you use `macos-setup.sh` or not, you should also install:
+
+- https://github.com/pyenv/pyenv[pyenv], at least 1.2.18
+- https://github.com/pypa/pipenv[pipenv], at least 2018.11.26
+
```

Comment:
Maybe we could also mention `nvm` and how to switch between versions that we need in the next section?

Reply:
Added this to the readme.